Developers/Linter_Bears_Advanced.rst

```diff
@@ -34,7 +34,7 @@ The path of the temporary configuration file can be accessed inside
         @staticmethod
```

Comment:
As a general rule of thumb, we avoid words like "Fix" in our commit message. They're often filler words that may hide the actual commit changes. I'd suggest changing the commit head to `Linter_Bears_Advanced: value=2 -> value2` as that's much more meaningful IMO.